### PR TITLE
feature: show modal when clicking "add fragment to collection" on item detail page

### PR DESCRIPTION
**Pull request by bertyhell**
Item toevoegen aan collectie

design:
https://drive.google.com/file/d/13X_OKKhwt_MU8xzJytpFoh5sYxucj7Hx/view

partially closes:
https://district01.atlassian.net/browse/AVO2-115

Still todo: 
* Get all collections from api
* Update collection to include new fragment
* Create new collection if selected by user
* Possibly add the option to cut the fragment (required flowplayer to be done)

Some of these require another this PR (https://github.com/viaacode/avo2-proxy/pull/12) to be merged, so making a PR of what i have already

Note: "eigen text" switch isn't required, since the user can just type in a new title/description if they want to.

### CODE REVIEW

File: src/item/views/Item.tsx [hunk 1]
```diff
@@ -358,14 +363,14 @@ const Item: FunctionComponent<ItemProps> = ({
...
 										</tr>
-										<tr>
-											<th scope="row">Klascement</th>
-											<td>
-												<a href={'http://www.klascement.be/link_item'}>
-													www.klascement.be/link_item
-												</a>
-											</td>
-										</tr>
+										{/*<tr>*/}
```
Comment (andrych) [inline src/item/views/Item.tsx]: Is dit iets dat achteraf nog geïmplementeerd moet worden of mag dit weg?

File: src/item/views/modals/AddFragmentToCollection.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#addFragmentToCollection {
```
Comment (andrych) [inline src/item/views/modals/AddFragmentToCollection.scss]: `.c-modal__body-add-fragment` or something maar geen id's

File: src/item/views/modals/AddFragmentToCollection.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+	return (
+		<Modal
+			title="Voeg fragment toe aan collectie"
+			size="auto"
+			scrollable={true}
+			isOpen={isOpen}
+			onClose={onClose}
+		>
+			<ModalBody>
+				<div id="addFragmentToCollection">
```
Comment (andrych) [inline src/item/views/modals/AddFragmentToCollection.tsx]: `<div className="c-modal__body-add-fragment">` zie comment hierboven

File: src/item/views/modals/AddFragmentToCollection.tsx [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+									<video
+										src={`${itemInfo.thumbnail_path.split('/keyframes')[0]}/browse.mp4`}
+										placeholder={itemInfo.thumbnail_path}
+										style={{ width: '100%', display: 'block' }}
+										controls={true}
+									/>
+								</Column>
+								<Column size="2-5">
+									<FormGroup label="Title">
+										<TextInput id="title" value={itemInfo.dc_title} />
```
Comment (andrych) [inline src/item/views/modals/AddFragmentToCollection.tsx]: TS error `Property 'dc_title' does not exist on type 'ItemResponse'`

File: src/item/views/modals/AddFragmentToCollection.tsx [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+									</FormGroup>
+									<FormGroup label="Beschrijving">
+										{/* TODO replace with component TextArea once built */}
+										<textarea id="description" value={itemInfo.dcterms_abstract} />
```
Comment (andrych) [inline src/item/views/modals/AddFragmentToCollection.tsx]: TS error `Property 'dcterms_abstract' does not exist on type 'ItemResponse'`

File: src/item/views/modals/AddFragmentToCollection.tsx (hunk 4 above)
Comment (bertyhell) [reply src/item/views/modals/AddFragmentToCollection.tsx]: this is already fixed in the other PR: https://github.com/viaacode/avo2-client/pull/40

File: src/item/views/modals/AddFragmentToCollection.tsx (hunk 5 above)
Comment (bertyhell) [reply src/item/views/modals/AddFragmentToCollection.tsx]: this is already fixed in the other PR: https://github.com/viaacode/avo2-client/pull/40

File: src/item/views/Item.tsx (hunk 1 above)
Comment (bertyhell) [reply src/item/views/Item.tsx]: Dit moet nog geimplementeerd worden, maar VIAA moet nog bepalen van waar die info moet komen. Voorlopig op backlog geplaatst: https://district01.atlassian.net/browse/AVO2-463

File: src/item/views/modals/AddFragmentToCollection.tsx (hunk 4 above)
Comment (andrych) [reply src/item/views/modals/AddFragmentToCollection.tsx]: Alright, I've approved PR #40

File: src/item/views/modals/AddFragmentToCollection.tsx (hunk 5 above)
Comment (andrych) [reply src/item/views/modals/AddFragmentToCollection.tsx]: Alright, I've approved PR #40

Comment (bertyhell) [reply src/item/views/modals/AddFragmentToCollection.tsx]: These should be fixed now

**Pull request merged**